first-analysis-steps/davinci.md

```diff
@@ -85,30 +85,55 @@ During this run, DaVinci didn't do anything: We didn't specify any algorithms to
 Usually, you will write an option file (e.g. `options.py`) and specify it as an argument to `gaudirun.py`:
 
 ```bash
-lb-run DaVinci/v42r6p1 gaudirun.py options.py
+lb-run DaVinci/v44r6 gaudirun.py options.py
 ```
 
 An `option.py` is just a regular Python script that specifies how to set things up in the software.
 Many of the following lessons will teach you how to do something with DaVinci by showing you how to write or extend an `options.py`.
 You can use the above command to test it.
 You can also specify several option files like this:
 ```bash
-lb-run DaVinci/v42r6p1 gaudirun.py options1.py options2.py
+lb-run DaVinci/v44r6 gaudirun.py options1.py options2.py
 ```
 They will then both be used to set up DaVinci.
 
 Do you want to get an overview of which versions of DaVinci exist? Use
 ```bash
 lb-run --list DaVinci
 ```
+
+{% callout "Which version of DaVinci should I use?" %}
+
+All available versions of DaVinci are given on the [DaVinci releases][releases]
+page. Which one should you use? They are a couple of guidelines to follow:
+
+1. When starting a new analysis, use the latest version suitable for the data
+   you would like to analysis.
+2. When continuing an analysis, use the same version of DaVinci consistently
+   throughout. This includes cases where you want to repeat an existing
+   analysis but with modified settings, e.g. re-running a Stripping line.
+
+There are a set of DaVinci versions for Upgrade studies (versions v50r0 and
```

Comment:
I thought about this and put v50r0 because this is where the upgrade versions do start. But you're right that this specific version is now obsolete. I'll change it to avoid potential confusion.